rts/lra/lra-client/src/main/java/io/narayana/lra/client/NarayanaLRAClient.java

```diff
@@ -569,9 +562,14 @@ public LRAStatus getStatus(URI uri) throws WebApplicationException {
         }
 
         try {
-            response = getTarget().path(getLRAId(lraId.toString())).path("status")
-                    .request()
-                    .get();
+            response = coordinatorClient.getLRAStatus(getLRAId(lraId.toString()));
+        } catch (WebApplicationException wae) {
+            Response waeResponse = wae.getResponse();
+            if (waeResponse.getStatus() == Response.Status.NOT_FOUND.getStatusCode()) {
+                String responseContent = waeResponse.readEntity(String.class);
+                throw new NotFoundException("Failed to get status of LRA id " + lraId
```

Comment:
For internationalization purposes we are used to use the error messages defined in `i18Logger` (ie. `LRALogger.i18NLogger.*`). Would be possible to change it that way?

(I understand that it was not done so in the code before but as I hit this in the diff I think it could be good to be changed.)

Reply:
I would suggest a new JBTM for this as this is not affecting only this code.

Reply:
+1